internal/compiler/widgets/cupertino-base/scrollview.slint

```diff
@@ -113,6 +115,15 @@ export component ScrollView {
         maximum:  i-flickable.viewport-height - i-flickable.height;
         page-size:  i-flickable.height;
         visible: i-flickable.viewport-height > i-flickable.height;
+
+        scroll-event(event) => {
+            if (event.delta-y != 0) {
+                root.viewport-y = max(root.visible-height - root.viewport-height, min(0px, root.viewport-y + event.delta-y));
```

Comment:
Won't this break the existing two-way binding in line 98/100? I have the feeling that this logic belongs into `ScrollBar` itself and then should modify `value` there.

Reply:
Yes you're right. Thank you. I've changed it.